lib/3scale_toolbox/crds/application_plan_dump.rb

```diff
@@ -0,0 +1,19 @@
+module ThreeScaleToolbox
+  module CRD
+    module ApplicationPlan
```

Comment:
Should this be called  `ApplicationPlanSerializer` to follow naming convention?

Reply:
done